db/migrate/20190509142148_update_custom_button_sets.rb

```diff
@@ -0,0 +1,21 @@
+class UpdateCustomButtonSets < ActiveRecord::Migration[5.2]
```

Comment:
This needs to be `[5.0]` as manageiq doesn't support 5.2 yet.

We're going to move this repo back to 5.0 to avoid this until then.

Reply:
@carbonin corrected. Thank you :+1: